localstack/services/sqs/provider.py

```diff
@@ -669,6 +671,12 @@ def create_queue(
 
         return CreateQueueResult(QueueUrl=queue.url(context))
 
+    def _verify_valid_queue_name(self, queue_name: str):
+        name_re = re.compile(sqs_constants.QUEUE_NAME_REGEX + "{" + f"{len(queue_name)}" + "}")
+        if not queue_name or len(queue_name) > 80 or name_re.match(queue_name) is None:
+            error_msg = "Can only include alphanumeric characters, hyphens, or underscores. 1 to 80 in length"
+            raise InvalidParameterValue(error_msg)
```

Comment:
we actually already have a method for this: https://github.com/localstack/localstack/blob/44ff8c78f3864c694dc52a44e0bfad032105be0f/localstack/services/sqs/provider.py#L105-L119